packages/itensor/quri_parts/itensor/estimator.py

```diff
@@ -56,41 +58,68 @@ def _estimate(operator: Estimatable, state: ITensorStateT) -> Estimate[complex]:
     op = convert_operator(operator, s)
 
     # calculate expectation value
-    psi = jl.apply(circuit, psi)
+    psi = jl.apply(circuit, psi, **kwargs)
     exp: float = jl.expectation(psi, op)
 
-    return _Estimate(value=exp, error=0.0)
+    # See https://github.com/QunaSys/quri-parts/pull/203#discussion_r1329458816
+    error = 0.0
+    if "maxdim" in kwargs or "cutoff" in kwargs:
+        error = np.nan
 
+    return _Estimate(value=exp, error=error)
 
-def create_itensor_mps_estimator() -> QuantumEstimator[ITensorStateT]:
-    """Returns a :class:`~QuantumEstimator` that uses ITensor MPS simulator to
-    calculate expectation values."""
 
-    return _estimate
+def create_itensor_mps_estimator(
+    *,
+    maxdim: Optional[int] = None,
+    cutoff: Optional[float] = None,
+    **kwargs: Any,
+) -> QuantumEstimator[ITensorStateT]:
+    """Returns a :class:`~QuantumEstimator` that uses ITensor MPS simulator to
+    calculate expectation values.
+    The following parameters including keyword
+    arguments `**kwargs` are passed to `ITensors.apply
+    <https://itensor.github.io/ITensors.jl/dev/MPSandMPO.html#ITensors.product-Tuple{ITensor,%20ITensors.AbstractMPS}>`_.
```

Comment:
My last suggestion was not correct…
What I wanted is to insert blank line between first and second sentence, without no whitespaces.
```suggestion
    """Returns a :class:`~QuantumEstimator` that uses ITensor MPS simulator to
    calculate expectation values.

    The following parameters including keyword
    arguments `**kwargs` are passed to `ITensors.apply
    <https://itensor.github.io/ITensors.jl/dev/MPSandMPO.html#ITensors.product-Tuple{ITensor,%20ITensors.AbstractMPS}>`_.
```

Reply:
👌